pkg/nanopb/nanopb.inc.mk

```diff
@@ -0,0 +1,22 @@
+PROTOC ?= protoc
+PROTOC_GEN_NANOPB ?= protoc-gen-nanopb
```

Comment:
Why not

`PROTOC_GEN_NANOPB ?= $(PKGDIRBASE)/nanopb/generator/protoc-gen-nanopb`

and don't have the user clone the repo a second time?